src/Thorn_Calculus-Core_Bisimilarities.thy

```diff
@@ -1325,6 +1325,140 @@ qed
 
 end
 
+text \<open>
+  We extend \<^theory_text>\<open>natural_simps\<close> with rules for eliminating duplicates of \<open>\<triangleright>\<^sup>\<infinity>\<close>-processes, which are
```

Comment:
Shouldn't it be `thorn_simps` instead of `natural_simps`?

Reply:
Yes, it should. :slightly_smiling_face: This bug existed already in [network-equivalences](https://github.com/input-output-hk/network-equivalences), so fixing it is actually not a topic of this pull request. That said, I’ve fixed it anyway, in order to reduce bureaucracy. The fix is in e20d2828c0bc32dca6000cebe18f3080394dea22.